includes/class-wp-vote-settings.php

```diff
@@ -92,19 +169,80 @@ function( $post_type ) {
 			function( $post_type ) {
 				$formated        = new \stdClass();
 				$formated->label = $post_type->label;
-				$formated->name  = $post_type->name;
 				return $formated;
 			},
 			$object_types
 		);
 
 		// As comment as an option.
 		$comment                  = new \stdClass();
-		$comment->label           = 'Comments';
-		$object_types['comments'] = $comment;
-
-		\UBC\CTLT\WPVote\Helpers\write_log( $object_types );
+		$comment->label           = 'Comment';
+		$object_types['comment'] = $comment;
 
 		return $object_types;
 	}
+
+	/**
+	 * Get a list of public available rubrics.
+	 *
+	 * @return array
+	 */
+	public static function get_rubrics_options() {
+		$args    = array(
+			'numberposts' => -1,
```

Comment:
`-1` in a query always concerns me. Just in case of a runaway query which creates loads of rubrics by accident or something like that. I'd much prefer we set a limit here. Something relatively sensible like 30. And then run that magic number through a filter. So that if someone needs to increase this number, we can add a separate plugin which increases it for those sites specificially. i.e.

`numberposts` => apply_filters( 'ubc_wp_vote_get_rubrics_options_numberposts', 30 );`

Likewise with other queries where we use -1.